source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+---
+title: Data Protection for GOV.UK Account users
+weight: 32
+last_reviewed_on: 2021-02-02
+review_in: 6 months
+---
+
+# Respond to data protection requests
+
+When we use and hold user data, users gain rights under the UK General Data Protection Regulation (UK GDPR).
+Users make requests to exercise those rights and we have responsibilities to respond.
+
+Below are some of the scenarios we will have to to understand and respond to along with information on who
+has responsibilities.
+
+Responding to requests may involve the GOV.UK Accounts team, GOV.UK User support and the GDS Privacy office.
+
+This guide is not an complete list of everything we must do under the UK GDPR. Instead it focuses on user rights
+requests and how to respond.
+
+# Who handles requests?
+
+Responding to rights requests is a collaboration between the Privacy Office, User support team and GOV.UK Accounts team.
+All requests should initially arrive through zendesk where they can be flagged by User Support.
+
+If a request arrives directly with the accounts team they should create a new zendesk ticket. Where possible the ticket should contain:
```

Comment:
How would a ticket come to the Account team without using Zendesk? I thought the feedback form created a Zendesk ticket?

Reply:
There are a few routes, none of them are well adversied - or preferred. But requests are still valid irrespective of where they arrive from.

I'll DM you a few examples.